CoreErrorReportView/src/au/gov/asd/tac/constellation/views/errorreport/ErrorReportTopComponent.java

```diff
@@ -645,12 +719,31 @@ public void updateSessionErrorsBox(final int insertPos) {
                 final int errCount = sessionErrors.size();
                 // rebuild                
                 sessionErrorsBox.getChildren().clear();
+                
+                // check popup selection
+                List<String> choices = new ArrayList<>();
+                if (getParams().hasParameter(REPORT_SETTINGS_PARAMETER_ID)) {
+                    MultiChoiceParameterValue multiChoiceValue = getParams().getMultiChoiceValue(REPORT_SETTINGS_PARAMETER_ID);
+                    choices = multiChoiceValue.getChoices();
+                }
```

Comment:
Is the value of `choices` supposed to be used somewhere?
If not, this block can be removed, as well as the variable definition.